xbmc/addons/gui/skin/SkinTimerManager.cpp

```diff
@@ -217,33 +202,25 @@ void CSkinTimerManager::Stop()
   m_timers.clear();
 }
 
-void CSkinTimerManager::StopThread(bool bWait /*= true*/)
-{
-  std::unique_lock<CCriticalSection> lock(m_timerCriticalSection);
-  m_bStop = true;
-  CThread::StopThread(bWait);
-}
-
 void CSkinTimerManager::Process()
 {
-  while (!m_bStop)
+  if (m_timers.empty())
+    return;
```

Comment:
I think this is not needed. The for loop will check this as well and will do nothing if no timers are available.

Reply:
true, container is empty which is pretty much the same thing